debian/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch
+MAINTAINER Shaun Jackman <sjackman@gmail.com>
+
+# Set apt in non-interactive mode during the image build
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Update debian package list
+RUN apt-get update
+
+# Set up UTF-8
+RUN apt-get install -y apt-utils
+RUN apt-get install -y locales
```

Comment:
The warning is:
`debconf: delaying package configuration, since apt-utils is not installed`
There is an extended discussion about this [here](https://github.com/phusion/baseimage-docker/issues/319). This warning is apparently harmless. By installing apt-utils, the warning appears only one time. It will be probably corrected in a future debian/stretch image.

https://github.com/Linuxbrew/docker/pull/8#discussion_r127991907